.github/workflows/e2e-tests.yaml

```
@@ -52,11 +61,27 @@ jobs:
             CSI_IMAGE_REPOSITORY=ghcr.io/${{ github.repository }} \
             ADDITIONAL_ARGS="--junit-report=./test-results/e2e-tests-results.xml"
 
+      - name: Copy S3 logs to artifacts directory
+        if: always()
+        run: |
+          mkdir -p artifacts/logs/s3
+          cp -r .github/scality-storage-deployment/logs/s3/* artifacts/logs/s3/ 2>/dev/null || true
+
+      - name: Upload S3 logs to Scality artifacts
+        if: always()
+        uses: scality/action-artifacts@v4
+        with:
+          method: upload
+          url: https://artifacts.scality.net
+          user: ${{ secrets.ARTIFACTS_USER }}
+          password: ${{ secrets.ARTIFACTS_PASSWORD }}
+          source: artifacts
```

I'm not a big fan of our own artifacts, especially considering the use of it on a open source repo, so I would suggest using GitHub's artifact in this case:

```suggestion
      - name: Upload S3 logs
        if: always()
        uses: actions/upload-artifact@v4
        with:
          name: e2e-tests-${{ matrix.ring_version }}
          path: artifacts
```

---

Changed in https://github.com/scality/mountpoint-s3-csi-driver/pull/61/commits/dc0b08b5d2441beb41a77fd1a4ebdaf2b94d570e